### DISCUSSION TITLE
some function not working in new leafmap version

### DISCUSSION THREAD

**Discussion by shi093** [opengeos/leafmap#124] [Q&A] [answered]
Under version 0.3.3, I can add layers using the menu on the top right corner (e.g., open GeoJson file by navigating to the file location), but this feature is not working under version 0.5.3. 
Also, this layer control button is not available in version 0.5.3. It seems that somethings have changed in the newer version.
<img width="84" alt="Screen Shot 2021-10-19 at 8 53 56 PM" src="https://user-images.githubusercontent.com/25441163/138010214-3237075d-6987-43f5-98d7-f88791a43e4a.png">

**Comment by giswqs** [comment, marked as answer]
This open vector bug has been fixed. Please update the package using `geemap.update_package()` and restart the kernel to take effect.

To show ipyleaflet's built-in layer control, use `m = geemap.Map(layer_ctrl=True)`

![image](https://user-images.githubusercontent.com/5016453/138015157-743f6274-2e37-4902-bf0e-124d67b6f0a7.png)

**Comment by shi093** [comment]
Thanks for replying. But the problem I reported are for the leafmap package, not geemap? or it doesn't matter?

**Comment by giswqs** [comment]
I just fixed it. Update leafmap using `leafmap.update_package()`. 

To use ipyleaflet's built-in layer control, use `m = leafmap.Map(layers_control=True)`

![image](https://user-images.githubusercontent.com/5016453/138016104-642df7ad-b93c-4a8c-8a7d-37b4414796c9.png)

**Comment by shi093** [comment]
Great! the add layers feature is now working properly both on my local computer and on Google Colab. The missing filed in the information box that I complained earlier was also fixed somehow. 
However, the layer control only works on Google Colab, not on my local computer. How come?

**Comment by shi093** [comment]
Okay, never mind. I figured it out. I need to use "layers_control" instead of "layer_control" on my local computer. "s" matters.

**Reply by giswqs** [reply]
Leafmap tries to use the same name as ipyleaflet. It is called LayersControl in ipyleaflet.

https://ipyleaflet.readthedocs.io/en/latest/api_reference/layers_control.html
